distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,363 @@
+
+.Catalog Architecture Diagram: Data
+[ditaa, catalog_architecture_data, png, ${image-width}]
+....
++------------------------------------------------------------+
+|                /-------------------\                       |
+|                |cDEFEndpoints      |                       |
+|                +------------+------+                       |
+|                |cDEF        |c369  |                       |
+|                | Operations | Data |                       |
+|/---------------+------------+------+------------+---------\|
+||cDEF           |cDEF               |cDEF        |cDEF     ||
+||  Transformers |                   | Federation | Sources ||
+|+---------------+ Catalog Framework +------------+---------+|
+||cDEF           |                   |cDEF   Eventing       ||
+||   Catalog     |                   +------------+---------+|
+||   Plugins     |                   |cDEF   Resources      ||
+|\---------------+-------------------+----------------------/|
+|                |cDEF               |                       |
+|                | Catalog Provider  |                       |
+|                +-------------------+                       |
+|                |cDEF               |                       |
+|                | Storage Provider  |                       |
+|                \-------------------/                       |
++------------------------------------------------------------+
+....
+
+The Catalog stores and translates Metadata, which can be transformed into many data formats, shared, and queried.
+The primary form of this metadata is the metacard. 
+A `Metacard` is a container for metadata. 
+`CatalogProviders` accept `Metacards` as input for ingest, and `Sources` search for metadata and return matching `Results` that include `Metacards`.
+
+=== Metacards
+
+A metacard is a single instance of metadata in the Catalog (an instance of a metacard type) which generally contains metadata providing a title for a product and describing that product's geo-location, created and modified dates, owner or producer, security classification, etc. 
+
+==== Metacard Type
+
+A metacard type indicates the attributes available for a particular metacard.
```

Comment:
Recommend providing an example here, e.g.:
`A metacard type indicates the attributes available for a particular type of data. For example, an image may have different attributes than a PDF document, so each could be defined to have their own metacard type.` 

For context, may also be helpful to reference the known Metacard Types that are already defined in DDF.